### PR TITLE
Division Table Upgrade

### PR DESCRIPTION
**Pull request by bqi343**
- add most remaining usaco tags (by find & replace)
- link to contest in division table
- make sure single problems in `problems` are processed correctly

### CODE REVIEW

File: content/4_Gold/BFS.mdx [hunk 1]
```diff
@@ -12,7 +12,9 @@ frequency: 2
 <!-- NOTE: general problems were RANDOMLY STARRED. please review! -->
 
 export const problems = {
-	sampleNormal: new Problem('CSES', 'Message Route', '1667', 'Easy', false, ['BFS']),
+	sampleNormal: new Problem('CSES', 'Message Route', '1667', 'Easy', false, [
+		'BFS',
```
Comment (HarryWangATX) [inline content/4_Gold/BFS.mdx:16]: was this an accidental misenter?

**Review by HarryWangATX — COMMENTED** [review]
will finish the rest later

File: content/4_Gold/BFS.mdx (hunk 1 above)
Comment (bqi343) [reply content/4_Gold/BFS.mdx:16]: this just happens when the last argument in constructor is a list 🤷

Comment (HarryWangATX) [reply content/4_Gold/BFS.mdx]: i c

File: content/4_Gold/BFS.mdx [hunk 2]
```diff
@@ -72,9 +97,17 @@ export const problems = {
 			['BFS'],
 			'usaco-695'
 		),
-		new Problem('Gold', "Bessie's Dream", '575', 'Hard', false, ['BFS']),
-		new Problem('Gold', "Replication", '1065', 'Hard', false, ['BFS']),
-		new Problem('Gold', 'A Pie for a Pie', '765', 'Very Hard', false, [], ''),
+		new Problem('Gold', "Bessie's Dream", '575', 'Hard', false, ['BFS'], ''),
```
Comment (HarryWangATX) [inline content/4_Gold/BFS.mdx:100]: fix formatting here??

File: content/4_Gold/BFS.mdx [hunk 3]
```diff
@@ -72,9 +97,17 @@ export const problems = {
...
+		new Problem('Gold', 'Replication', '1065', 'Hard', false, ['BFS'], ''),
```
Comment (HarryWangATX) [inline content/4_Gold/BFS.mdx:101]: and here

File: content/4_Gold/DSU.mdx [hunk 4]
```diff
@@ -8,61 +8,67 @@ description: 'The Disjoint Set Union (DSU) data structure allows you to add edge
 frequency: 3
 ---
 
-
-
 export const problems = {
-	sample: [new Problem('YS', 'Union Find', 'unionfind', 'Easy', false, [])],
+	sample: [
+		new Problem('YS', 'Union Find', 'unionfind', 'Easy', false, ['DSU'], ''),
```
Comment (HarryWangATX) [inline content/4_Gold/DSU.mdx:13]: here

File: content/4_Gold/DSU.mdx [hunk 5]
```diff
@@ -8,61 +8,67 @@ description: 'The Disjoint Set Union (DSU) data structure allows you to add edge
...
+			['DSU'],
+			'usaco-992'
 		),
-		new Problem('Silver', 'Wormhole Sort', '992', 'Easy', true, [
-			'DSU',
-		], 'usaco-992'),
-		new Problem('Gold', 'Moocast', '669', 'Easy', false, [
-			'DSU',
-		], 'usaco-669'),
+		new Problem('Gold', 'Moocast', '669', 'Easy', false, ['DSU'], 'usaco-669'),
```
Comment (HarryWangATX) [inline content/4_Gold/DSU.mdx:44]: here

File: content/4_Gold/Faster_Hash.mdx [hunk 6]
```diff
@@ -8,12 +8,20 @@ prerequisites:
   - unordered
 ---
 
-
-
 export const problems = {
-	three: [new Problem('Gold', '3SUM', '994', 'Normal', false, [], '')],
+	three: [
+		new Problem('Gold', '3SUM', '994', 'Normal', false, ['Prefix Sums'], ''),
```
Comment (HarryWangATX) [inline content/4_Gold/Faster_Hash.mdx:13]: here

**Review by HarryWangATX — CHANGES_REQUESTED** [review]
finish more later so long...

File: content/4_Gold/BFS.mdx (hunk 2 above)
Comment (bqi343) [reply content/4_Gold/BFS.mdx:100]: I think it's fine if auto-formatter puts this all on a single line.

File: content/4_Gold/Knapsack_DP.mdx [hunk 7]
```diff
@@ -266,7 +279,7 @@ public class Main {
 
 </JavaSection>
 
-<PySection/>
+<PySection />
```
Comment (HarryWangATX) [inline content/4_Gold/Knapsack_DP.mdx:282]: uh oh...

Comment (HarryWangATX) [reply content/4_Gold/Knapsack_DP.mdx:282]: wait nvm i didnt notice this ones empty

File: content/6_Advanced/Springboards.mdx [hunk 8]
```diff
@@ -8,20 +8,28 @@ description: 'A solution to USACO Gold - Springboards.'
 frequency: 1
 ---
 
-
-
 export const problems = {
-	sample: [new Problem('Gold', 'Springboards', '995', 'Hard', false, [], '')],
+	sample: [
+		new Problem('Gold', 'Springboards', '995', 'Hard', false, ['PURQ'], ''),
```
Comment (HarryWangATX) [inline content/6_Advanced/Springboards.mdx:13]: here

File: solutions/1_extra_usaco_probs.ts [hunk 9]
```diff
@@ -6,6 +6,34 @@ import { Problem } from '../content/models';
...
+  new Problem(
+    'Plat',
+    'Greedy Gift Takers',
+    '770',
+    'Normal',
+    false,
+    ['Binary Search'],
+    ''
+  ),
+  new Problem('Plat', 'Lifeguards', '792', 'Normal', false, ['DP'], ''),
```
Comment (HarryWangATX) [inline solutions/1_extra_usaco_probs.ts:27]: Didnt reformamt this

**Review by HarryWangATX — CHANGES_REQUESTED** [review]
i think that was it

File: solutions/1_extra_usaco_probs.ts (hunk 9 above)
Comment (bqi343) [reply solutions/1_extra_usaco_probs.ts:27]: not an issue, see above comment

**Pull request merged**